config/initializers/stripe.rb

```diff
@@ -0,0 +1 @@
+Stripe.api_key = ENV['STRIPE_SECRET_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.